resources/views/tools/show.blade.php

```diff
@@ -0,0 +1,111 @@
+<x-app-layout body-class="page tool" page-width="wide" header-class="full header--tool">
+    <x-slot name="title">{{ $tool->title }}</x-slot>
+    <x-slot name="header">
+        <div class="center center:wide welcome pb-32">
+            <ol class="breadcrumbs" role="list">
+                <li><a href="{{ localized_route('tools.index') }}">{{ __('Tools') }}</a></li>
+            </ol>
+            <h1>
+                {{ $tool->title }}
+            </h1>
+            <x-interpretation name="{{ __('Tool', [], 'en') }}" />
+
+            @if ($tool->description)
+                {!! Str::markdown($tool->description, config('markdown')) !!}
+            @endif
+        </div>
+    </x-slot>
+
+    <div class="center center:wide pb-28">
+        <div class="stack md:w-2/3">
+            @if (empty($content) || $content->isEmpty())
+                <p>{{ __('Coming soon') }}</p>
+            @else
+                {{ $content }}
+            @endif
+        </div>
+    </div>
+    @if ($tool->documents->count())
+        <x-section class="full dark -mb-8" aria-labelledby="download-tool">
+            <div class="center stack stack:xl py-20" x-data="{
+                open: false,
+                email: @auth() '{{ Auth::user()->email }}' @else false @endauth
+            }">
+                <h2 class="text-center" id="download-tool">{{ __('Download tool') }}</h2>
+                <x-interpretation name="{{ __('Download tool', [], 'en') }}" />
+                <div class="field" x-show="email === false">
+                    <x-hearth-label for="email" :value="__('Your email (optional)')" />
+                    <x-hearth-hint
+                        for="email">{{ __('[Copy about why you’re collecting their email]') }}</x-hearth-hint>
+                    <x-hearth-input name="email" type="email" x-ref="email" hinted />
+                    <button class="w-full" type="button"
+                        @click="email = $refs.email.value ? $refs.email.value : null">{{ __('Get download links') }}</button>
+                </div>
+                <div class="stack stack:xl" x-show="email !== false">
+                    <p class="text-center">{{ __('Questions about the tool? Please contact [email here].') }}</p>
+                    <p class="text-center"><strong>{{ __('Your download links are below:') }}</strong></p>
+                    <ul role="list">
+                        @foreach ($tool->revisions->groupBy('date')->reverse()->first() as $revision)
+                            <li class="row flex items-center justify-between border-x-0 border-b border-t-0 border-solid py-3"
+                                style="border-block-end-color: var(--interactive);">
+                                <span>{{ $revision->document->name }}
+                                    {{ $revision->date->format('Y-m-d') }}.{{ pathinfo($revision->file, PATHINFO_EXTENSION) }}</span>
```

Comment:
It might be better to not show the file extension here as the English and French may be different file types. Also the name of the file will be different in French because of the appended language code in the name. This is particularly true when you are viewing the site in French as the name shown may not match either the French or English file name exactly.